From 9cbdf19e1964d8a7b7cc7829529528a13efe8a47 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 23 Jul 2015 19:39:42 +0000
Subject: [PATCH 184/462] Mantis 0037349: extension; integration of ingenico
 payment device for KIS kiosk

added variables DIR_GSOAP_2_7_17, WSDLCOMP_2_7_17, SOAPCOMP_2_7_17, SOAPCOMP_2_7_17_FLAGS for gSOAP 2.7.17

git-svn-id: svn://localhost/SelfServiceBuild/trunk@276 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index d37a165..6b01abf 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -346,6 +346,11 @@ set DIR_GSOAP=%DIR_GLOBALSLIB%\gsoap\2.7.8\win32
 set WSDLCOMP=%DIR_GSOAP%\bin\wsdl2h.exe
 set SOAPCOMP=%DIR_GSOAP%\bin\soapcpp2.exe
 
+set DIR_GSOAP_2_7_17=%DIR_GLOBALSLIB%\gsoap\2.7.17\win32
+set WSDLCOMP_2_7_17=%DIR_GSOAP_2_7_17%\bin\wsdl2h.exe
+set SOAPCOMP_2_7_17=%DIR_GSOAP_2_7_17%\bin\soapcpp2.exe
+set SOAPCOMP_2_7_17_FLAGS=-I%DIR_GSOAP_2_7_17%\import 
+
 set DIR_LUA=%DIR_GLOBALSLIB%\lua\5.0.a
 set INC_LUA=-I%DIR_LUA%\include
 set LIB_LUA=%DIR_LUA%\win32\lib\lua50a.lib
-- 
2.41.0.windows.1

